.github/workflows/main.yml

```diff
@@ -723,6 +729,18 @@ jobs:
         rustup target add ${{ matrix.target }}
       if: matrix.target != ''
 
+    - run: rustup target add aarch64-linux-android
+    - run: rustup target add x86_64-linux-android
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v2
+    - uses: actions/cache@v3
+      with:
+        path: ${{ runner.tool_cache }}/cargo-ndk
+        key: cargo-ndk-bin-${{ env.CARGO_NDK_VERSION }}
+    - run: echo "${{ runner.tool_cache }}/cargo-ndk/bin" >> $GITHUB_PATH
+    - run: cargo install --root ${{ runner.tool_cache }}/cargo-ndk --version ${{ env.CARGO_NDK_VERSION }} cargo-ndk
```

Comment:
Similar to above, could this happen only for android targets? (avoids installing it for all "build" builders)